packages/appeals-service-api/src/repositories/sql/appeal-user-repository.js

```diff
@@ -0,0 +1,20 @@
+const dbClient = require('../../db/db-client');
+
+class AppealUserRepository {
+	constructor() {}
+	/**
+	 * Get a user by email
+	 *
+	 * @param {string} email
+	 * @returns {import('@prisma/client').AppealUser}
```

Comment:
This is a promise, so should return `Promise<...type...>`.

Also, won't this return null or something if no user is found?